src/main/conversions.c

```diff
@@ -3034,3 +3056,145 @@ const char *convert_pyobject_to_str(PyObject *py_obj)
 error:
     return NULL;
 }
+
+static PyObject *
+create_py_node_stats_from_as_node_stats(as_error *error_p,
+                                        as_node_stats *node_stats)
+{
+    PyObject *py_node_stats = create_class_instance_from_module(
+        error_p, "aerospike_helpers.metrics", "NodeStats", NULL);
+    if (!py_node_stats) {
+        return NULL;
+    }
+
+    bool success =
+        py_obj_set_common_attrs_from_as_node(py_node_stats, node_stats->node);
+    if (!success) {
+        goto error;
+    }
+
+    success = py_obj_set_common_attrs_from_as_node_stats(error_p, py_node_stats,
+                                                         node_stats);
```

Comment:
nit: extra empty line

Reply:
I tried to remove the extra line but clang-format is forcing `node_stats);`.. we'd need to change `.clang-format` so this doesn't happen

Reply:
Ok, nbd